velox/functions/sparksql/Arithmetic.h

```diff
@@ -47,6 +47,23 @@ struct RemainderFunction {
   }
 };
 
+template <typename T>
+struct RemainderFloatFunction {
+  template <typename TInput>
+  FOLLY_ALWAYS_INLINE bool
+  call(TInput& result, const TInput a, const TInput n) {
+    if (UNLIKELY(n == 0)) {
+      return false;
+    }
+    if (UNLIKELY(n == 1 || n == -1)) {
+      result = 0;
+    } else {
+      result = std::fmod(a, n);
```

Comment:
Looks like we can combine this implementation with RemainderFunction and reuse the other part of code with below.
```
if constexpr (std::is_same_v<TInput, float> || std::is_same_v<TInput, double>) {
  result = std::fmod(a, n);
} else {
  result = a % n;
}
```

Reply:
thanks!  Fixed